docs/contributors/code/release.md

```diff
@@ -140,6 +148,8 @@ The method for point releases is nearly identical to the main Plugin release pro
 
 The point release should only contain the _specific commits_ required. To do this you should checkout the previous _minor_ stable (i.e. non-RC) release branch (e.g. `release/12.5`) locally and then cherry pick any commits that you require into that branch.
 
+**Important**: you must assign the correct Milestone to each PR using the Github GUI. If the Milestone is closed then you will need to re-open it.
```

Comment:
I suggest we do two things here:
1. omit the `closed` Milestone case as I think it's an edge case and as we discussed in DM, and I don't even remember the error I had encountered - it has happened to me once in the past. So it maybe adds a bit more noise to the docs. Alternatively we should add it below in trouble shooting.
2. Regarding the assignment I think we should say that when a PR is merged is automatically assigned a milestone for the next stable release, so we would need to remove that milestone and add the previous release one. Example a PR that needs to cherry picked for a patch release would have milestone : `Gutenberg 12.6` and we should change it to have only `Gutenberg 12.5`. Also we need to say that this is needed because the changelog is automatically created based on the `milestone`.

Reply:
>  Alternatively we should add it below in trouble shooting.

Let's do that. The more edge cases we have covered the better IMHO. But I agree I don't want to add undue complexity or noise to the main body of the documentation prose. 

> Regarding the assignment I think we should say that when a PR is merged is automatically assigned a milestone for the next stable release, so we would need to remove that milestone and add the previous release one. Example a PR that needs to cherry picked for a patch release would have milestone : Gutenberg 12.6 and we should change it to have only Gutenberg 12.5. Also we need to say that this is needed because the changelog is automatically created based on the milestone.

This is a great point. Being clearer here will really help. Thanks!

Reply:
✅